spec/requests/users_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
```

Comment:
User Spec file should include tests for both `show` and `index` that ensures correct view template is rendered.
`Example : expect(response).to render_template(:show)`

User Spec file should include test for both `show` and `index` that ensures the response body includes the correct placeholder text.
`Example : expect(response.body).to include("Welcome to user index page")`